test/testlib/nuodb_database_utilities.go

```diff
@@ -593,6 +597,10 @@ func ServeFileViaHTTP(t *testing.T, namespaceName string, localFilePath string)
 }
 
 func GetNuoDBVersion(t *testing.T, namespaceName string, options *helm.Options) string {
+	_, ok := nuoDBVersionMap[options.SetValues["nuodb.image.tag"]]
+	if(ok) {
+		return nuoDBVersionMap[options.SetValues["nuodb.image.tag"]]
+	}
```

Comment:
Nit: You can avoid getting the value from the map twice. It is often written in Go like this:

```go
        if v, ok := nuoDBVersionMap[options.SetValues["nuodb.image.tag"]]; ok {
                 return v
        }
```

Reply:
Thanks. I will make the necessary changes.